yarn.lock

```diff
@@ -1681,34 +1681,6 @@
     uuid "^8.3.0"
     vfile "^4.2.0"
 
```

Comment:
The change below was actually added in the original Vue PR, so these lines were not here before. I guess they are not needed after all.
<img width="1898" alt="Screen Shot 2022-07-19 at 10 49 24 AM" src="https://user-images.githubusercontent.com/11838280/179816671-061a8610-00ed-4315-9bb6-7a6dfc7d1815.png">